src/server/utils/ride-helper.js

```diff
@@ -183,7 +183,9 @@ export const addRide = (ride, userId, origin, destination, cb) => {
   fetch(endpoint, {
       method: 'POST',
       headers: {
-        'Content-Type': 'application/json'
+        'Content-Type': 'application/json',
```

Comment:
Its not a duplicate. I just copy pasted what you and Shez told me to from your previous PR....its the exact same thing? Whats the problem?